public/app/features/plugins/pluginPreloader.ts

```diff
@@ -51,16 +62,16 @@ async function preload(config: AppPluginConfig): Promise<PluginPreloadResult> {
       isAngular: config.angular.detected,
       pluginId,
     });
-    const { extensionConfigs = [], exposedComponentConfigs = [] } = plugin;
+    const { extensionConfigs = [], exposedComponentConfigs = [], addedComponentConfigs } = plugin;
```

Comment:
**Nit:** we could add a default `= []` here for `addedComponents` as well, and then we wouldn't need the ` || []` on lin `48`. (It's not like this approach would be better, only to be consistent with the other configs.)

Reply:
Yep good shout! While at it, I'm making the addedComponentConfigs prop required https://github.com/grafana/grafana/pull/91877/commits/d8332dfecae3941cc758a203e1975cb14c97c160